curriculum/challenges/english/25-front-end-development/workshop-binary-search/683ec95f2f8781355556ff81.md

```diff
@@ -0,0 +1,65 @@
+---
+id: 683ec95f2f8781355556ff81
+title: Step 9
+challengeType: 20
+dashedName: step-9
+---
+
+# --description--
+
+You need to test out things so you will understand the flow of the algorithm at this initial stage.
+
+To do that, you need to first break out of the loop and return a custom message in the function, at least for now.
+
+That's because the current implementation will only allow one iteration, so if the condition is not met, there will be an infinite loop.
+
+Just after the `if`statement, use the `break` keyword to break out of the `while` loop. Then, after the `while` loop, return an empty list to signify that the value was not found.
+
+You will understand the purpose of this message in a few steps.
```

Comment:
```suggestion
To do that, you need to first break out of the loop. That's because the current implementation will only allow one iteration, so if the condition is not met, there will be an infinite loop.

Just after the `if`statement, use the `break` keyword to break out of the `while` loop. Then, after the `while` loop, return an empty list to signify that the value was not found.
```